### ISSUE TITLE
Lower-dimensional embedding layer

### ISSUE THREAD

**Issue by jeroneandrews** [ox-vgg/vgg_face2#23]
>  "<model-#D>" means that a lower-dimensional embedding layer is stacked on the top of the original final feature layer adjacent to classifier. The models are trained with standard softmax loss.

How do you learn these lower-dimensional embedding layers? In the pytorch code for the lower-dimensional embedding layers, there is no longer any reference to the 8631 training identities.

**Comment by WeidiXie** [comment]
we train the original ResNet50 first (ending up with 2048 dimension), then we attach another fc layer, which projects 2048 to 256 for instance, then fine-tune the model by doing classification, i.e. 256 --> 8631.

Best
Weidi

**Comment by jeroneandrews** [comment]
OK, that's what I thought, but I was just checking. Thanks for your quick response @WeidiXie
